core/installation/views/steps/admin_account_setup.php

```diff
@@ -118,8 +118,8 @@
 				<div class="sixteen wide mobile twelve wide tablet ten wide computer column">
 					<div class="ui form">
 						<?php
-							create_field('text', $language['username'], 'username', 'inputUsername');
-							create_field('email', $language['email_address'], 'email', 'inputEmail');
+							create_field('text', $language['username'], 'username', 'inputUsername', getenv('NAMELESS_ADMIN_USERNAME') ?: '');
```

Comment:
Apparently `getenv` returns false if the variable isn't set - do you know if this results in the string "false" being added to the field without the environment variable?

Reply:
`getenv('NAMELESS_ADMIN_USERNAME') ?: ''` is short for `getenv('NAMELESS_ADMIN_USERNAME') ? getenv('NAMELESS_ADMIN_USERNAME') : '' `. I'm pretty sure the string `"false"` evaluates to the boolean `false` in php?

Reply:
Sorry I didn't see `?:`, oops 😄